android/tooling/rib-intellij-plugin/build.gradle

```diff
@@ -32,6 +32,22 @@ templates {
         }
     }
 
+    template {
```

Comment:
maybe move above 0.3? the order in the dropdown will be the same as here

Reply:
done